helpers/Singular_Value_Calculator/SVD

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+import pandas as pd
+import matplotlib as plt
+import sympy as smp
+from sympy.solvers import solve
+from sympy import Symbol
+from sympy import *
```

Comment:
we should avoid importing ALL of sympy (using `*`) as it loads a lot of unnecessary things into the global scope and could impact performance by being greedy with memory.